lib/active_model/serializer.rb

```diff
@@ -17,32 +19,10 @@ class Serializer
     include Associations
     include Attributes
     include Caching
+    include Links
+    include Type
```

Comment:
Well it's quite a common pattern. In any case, it does not make sense to have a 300+ lines `serializer.rb` file.